bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/support/BusinessNetworkAwareInitiatedFlow.kt

```diff
@@ -0,0 +1,29 @@
+package net.corda.businessnetworks.membership.member.support
+
+import co.paralleluniverse.fibers.Suspendable
+import net.corda.businessnetworks.membership.common.CounterPartyNotAMemberException
+import net.corda.businessnetworks.membership.member.GetMembershipsFlow
+import net.corda.core.flows.FlowLogic
+import net.corda.core.flows.FlowSession
+import net.corda.core.identity.Party
+
+abstract class BusinessNetworkAwareInitiatedFlow<out T>(val flowSession: FlowSession) : FlowLogic<T>() {
+
+    @Suspendable
+    override fun call(): T {
+        confirmInitiatorIsAMemberOfThisBN(flowSession.counterparty)
```

Comment:
The same as above ^^. Can we simplify to `verifyMembership()` or something?

Reply:
ok